mne/viz/topo.py

```diff
@@ -798,7 +802,7 @@ def _plot_update_evoked_topo_proj(params, bools):
 
 
 def plot_topo_image_epochs(epochs, layout=None, sigma=0., vmin=None,
-                           vmax=None, colorbar=True, order=None, cmap='RdBu_r',
+                           vmax=None, colorbar=None, order=None, cmap='RdBu_r',
```

Comment:
You changed default value of colorbar in a public function but did not update its docstring. (see [code-guidelines](https://mne-tools.github.io/dev/contributing.html#code-guidelines), to know more about mne-docstring conventions).